app/scripts/models/account.js

```diff
@@ -350,7 +363,15 @@ define([
     }
   });
 
-  ['getAvatar', 'getAvatars', 'postAvatar', 'deleteAvatar', 'uploadAvatar']
+  [
+    'getProfile',
+    'getAvatar',
+    'getAvatars',
+    'postAvatar',
+    'deleteAvatar',
+    'uploadAvatar',
+    'postDisplayName'
```

Comment:
You added both `getDisplayName` and `postDisplayName` to the [profile-client](https://github.com/mozilla/fxa-content-server/pull/2748/files#diff-23f54373ee1fff72eefe1bbf757908a3R70), but only `postDisplayName` here, intentional?